__tests__/components/Comments/Comment.test.js

```diff
@@ -211,7 +218,8 @@ describe('Comment test should run correctly', () => {
         findCommentAndUpdate={findComment}
         updateVote={updateVote}
         time={time}
-      />
+      />,
+      {wrapper: Store}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the `Store` context provider is being used as a wrapper in multiple test render functions. Ensure that the `Store` context provider is correctly initialized with all necessary values for these tests.

```diff
-       />
+       />,
+       {wrapper: Store}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->